kytos/cli/commands/napps/api.py

```diff
@@ -159,6 +159,8 @@ def install_napp(cls, mgr):
                     log.error('  NApp not found.')
                 else:
                     log.error('  NApps Server error: %s', e)
+            except URLError as e:
+                log.error('  NApps Server error: '+str(e.reason))
```

Comment:
@macartur, please, fix this logging statement. This syntax (`+`) is not in compliance with our coding style guidelines.